packages/checkout-ui-extensions/src/extension-points/extension-points.ts

```diff
@@ -27,6 +27,10 @@ export interface ExtensionPoints {
     StandardApi<'Checkout::KitchenSink'>,
     AllComponents
   >;
+  'Checkout::LineItem::RenderAfter': RenderExtension<
```

Comment:
Can we keep this change private? I don't think developers can use it so it's going to be confusing for them.

Reply:
Do we need to? Ideally I'd like the package to be in sync with `checkout-web` otherwise it makes these copy-overs more difficult.

In any case, isn't this type necessary to make it possible to call `render('Checkout::LineItem::RenderAfter', ...)`?

Reply:
It is, I just think it's confusing for external consumers that they can call that, and it clearly should do a pretty obvious thing, but it won't do that obvious thing for them.